lib/SILGen/SILGen.h

```diff
@@ -318,6 +318,17 @@ class LLVM_LIBRARY_VISIBILITY SILGenModule : public ASTVisitor<SILGenModule> {
   /// Emit the self-conformance witness table for a protocol.
   void emitSelfConformanceWitnessTable(ProtocolDecl *protocol);
 
+  // SWIFT_ENABLE_TENSORFLOW
+  /// Emit the differentiability witness for the given original function
+  /// declaration and SIL function, parameter indices, and JVP and VJP
+  /// functions (null if undefined).
+  void emitDifferentiabilityWitness(AbstractFunctionDecl *originalAFD,
+                                    SILFunction *originalFunction,
+                                    IndexSubset *parameterIndices,
```

Comment:
Use a `const SILAutoDiffConfig &` parameter instead.

Reply:
Done in https://github.com/apple/swift/pull/27652/commits/fa71b2609d18a32ac158ae7b9e7e7f0dc013df90.